projects/VitalRecord/CodeSystems.cs

```diff
@@ -119,6 +119,9 @@ public static class CodeSystems
         /// <summary> Component </summary>
         public static string Component = "http://hl7.org/fhir/us/vrdr/CodeSystem/vrdr-component-cs";
 
+        /// <summary> Component </summary>
+        public static string ComponentCodeVR = "http://hl7.org/fhir/us/vr-common-library/2024Jan/CodeSystem-codesystem-vr-component";
```

Comment:
Should this be http://hl7.org/fhir/us/vr-common-library/CodeSystem/codesystem-vr-component ?

See IG https://hl7.org/fhir/us/vr-common-library/2024Jan/StructureDefinition-input-race-and-ethnicity-vr.html